tests/ui/terse-output-format-pass/expected

```diff
@@ -1,2 +1 @@
 VERIFICATION RESULT:
-** 0 of 3 failed
```

Comment:
This is now pretty much not doing any checking. Should we add this line back, or at least its prefix `** 0 of`?

Reply:
My interpretation was this test was mostly "run the output format at all" and not actively really looking for any results here.